examples/automated-event-reminders/main.bal

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/task;
+import ballerinax/discord;
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string[] scopes = ?;
+
+string guildId = "YOUR_GUILD_ID";
+
+# Represents a Discord Channel.
+public type Channel record {
+    # ID of the channel
+    string id;
+};
+
+# Represents a Discord Event.
+public type Event record {
+    # ID of the event
+    string id;
+    # Name of the event
+    string name;
+};
+
+discord:ConnectionConfig discordConfig = {
+    auth: {
+        clientId,
+        clientSecret,
+        scopes
+    }
+};
+
+discord:Client discord = check new (discordConfig);
+
+public function main() returns error? {
+    // Registers the scheduled event.
+    discord:guild_id_scheduledevents_body payload = {
+        name: "Community Meeting",
+        description: "Join us for our monthly community meeting!",
+        scheduled_start_time: "2024-12-31T23:59:59Z",
+        scheduled_end_time: "2024-12-31T23:59:59Z",
+        privacy_level: 2,
+        entity_type: 1,
+        entity_metadata: {
+            location: "Virtual"
+        }
+    };
+
+    Event|error eventResponse = discord->/guilds/[guildId]/scheduled\-events.post(payload);
+    if eventResponse is error {
+        log:printError("Error creating the scheduled event: ", eventResponse);
+        return;
+    }
```

Comment:
Can we do this ? I think here our focus is to show the usability of the discord connector. I think we can omit error handling part in this context.
```suggestion
    Event eventResponse = check discord->/guilds/[guildId]/scheduled\-events.post(payload);
```

Reply:
Yeah thats also an option. But at the same time these examples should represent how these connectors **should** be used in real world scenarios, isn't it? So I thought including proper error handling logic will guide the new users who wants to use the connector to follow the best practices. WDYT?

Reply:
In that case lets keep this +1